R/convert_ov_wenr.R

```diff
@@ -0,0 +1,37 @@
+#' Convert possible B_OV_WENR values to standardised values
+#' 
+#' This function converts numeric values for OV_WENR to values used by other OBIC functions if numeric values are entered.
+#' 
+#' @param B_OV_WENR () Data on soil compaction risk that may have to be converted to string
+#' 
+#' @import data.table
+#' 
+#' @export
+
+convert_WENR <- function(B_OV_WENR) {
+  # Create data table
+  dt <- data.table(B_OV_WENR = B_OV_WENR)
+  if(is.numeric(B_OV_WENR)){
```

Comment:
The formatting of this if statement looks a little bit weird. Would this be more clear:

```
    if(is.numeric(B_OV_WENR)) {
       checkmate::assert_subset(B_OV_WENR, empty.ok = FALSE, choices = c(1, 2, 3, 4, 5, 10, 11, 401, 901, 902))
   }
```